WTWSMS/history/titles/d_vasconic_pagan_reformed.txt

```diff
@@ -0,0 +1,4 @@
+475.1.1= {
```

Comment:
No it wouldn't. Plus, it might not cause the fix to work. I tried setting the Miaphysite fix to "37.1.1" and it didn't work. I had to put it under its first holder's history registry.

That entire thing is hidden, anyway.